pkg/image/validation/kubernetes.go

```diff
@@ -143,69 +142,115 @@ func validateManifestURLs(k8s *image.Kubernetes) []FailedValidation {
 	return failures
 }
 
-func validateHelmCharts(k8s *image.Kubernetes, imageConfigDir string) []FailedValidation {
+func validateHelm(k8s *image.Kubernetes, imageConfigDir string) []FailedValidation {
 	var failures []FailedValidation
 
-	if len(k8s.HelmCharts) == 0 {
+	if len(k8s.Helm.Charts) == 0 {
+		return failures
+	}
+
+	if len(k8s.Helm.Repositories) == 0 {
+		failures = append(failures, FailedValidation{
+			UserMessage: "Helm Charts defined with no Helm Repository defined.",
```

Comment:
What do you think about using lower case for all `Chart`, `Charts`, `Repository` occurrences?